packages/lpa-questionnaire-web-app/src/controllers/upload-plans.js

```diff
@@ -0,0 +1,11 @@
+const { VIEW } = require('../lib/views');
+const getAppealSideBarDetails = require('../lib/appeal-sidebar-details');
+
+exports.getUploadPlans = (req, res) => {
+  res.render(VIEW.UPLOAD_PLANS, {
+    appeal: getAppealSideBarDetails(req.session.appeal),
+    backLink: req.session.backLink || `/${req.params.id}/${VIEW.TASK_LIST}`,
+    maxFileSize: '15MB',
```

Comment:
This is more for thought that action.

One thing that chopped and changed during appeal submission upload work was the max file size. First it was a gig, then 10mb, then 50mb... and so on.

If I were to redo this now, I'd probably make this more configurable on a per upload basis. By which I mean supporting documents might be 10mb each, but decision file upload could be 50mb or whatever. 

As it stands, I generically implemented a single file size, which is fortunate in that it does meet their requirements, but obv isn't super customisable.

I made that value available as a nunjucks global so it was always kept in sync with whatever config value we set:

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/src/app.js#L88

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/src/config.js#L34

And then you can see an example of how that's used dotted around the code:

https://github.com/Planning-Inspectorate/appeal-planning-decision/search?q=uploadApplicationMaxFileSize

Just food for thought - maybe they will never change it on you. Nice to have just one place to change if they do.

Reply:
Thanks Chris. I brought this over and yet completely forgot about it when doing this work. Have amended in line with Team 1. I appreciate what you say about having the option for individual uploads, however we have been assured all file uploads will be the same. If not then we can make use of individual environment variables. For now I have used a single config item for the size, which we can expand later (we have a lot more upload questions in the questionnaire to come).